cli/ballerina-packerina/src/test/resources/test-resources/valid-project/src/mytemplate/tests/main_test.bal

```diff
@@ -33,7 +33,7 @@ function afterFunc () {
 
 # After Suite Function
 
-@test:AfterSuite
+@test:AfterSuite {}
 function afterSuiteFunc () {
     io:println("I'm the after suite function!");
 }
```

Comment:
Better to add a new line at the end there are some more files you need to fix.

Reply:
Added to necessary files in f6db6be